### PR TITLE
[zh] translate tasks/Enabling Topology Aware Hints

### LINKED ISSUE #27995: [zh] translate tasks/Enabling Topology Aware Hints
**This is a Feature Request**
translate /docs/tasks/administer-cluster/enabling-topology-aware-hints.md into Chinese.

**What would you like to be added**
/zh/docs/tasks/administer-cluster/enabling-topology-aware-hints.md

/language zh
/assign

### PR DESCRIPTION
**Pull request by zhiguo-lu**
translate /docs/tasks/administer-cluster/enabling-topology-aware-hints.md into Chinese.

close #27995

### CODE REVIEW

**Comment by zhiguo-lu** [comment]
https://deploy-preview-28155--kubernetes-io-master-staging.netlify.app/docs/tasks/administer-cluster/enabling-topology-aware-hints/

File: content/zh/docs/tasks/administer-cluster/enabling-topology-aware-hints.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+<!-- 
+_Topology Aware Hints_ enable topology aware routing with topology hints
+included in {{< glossary_tooltip text="EndpointSlices" term_id="endpoint-slice" >}}.
+This approach tries to keep traffic close to where it originated from;
+you might do this to reduce costs, or to improve network performance.
+-->
+_拓扑感知提示_ 启用具有拓扑提示能力的路由，该拓扑提示能力内置于
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/enabling-topology-aware-hints.md:25]:
```suggestion
_拓扑感知提示_ 启用具有拓扑感知能力的路由，其中拓扑提示信息包含在
```

**Review by chenrui333 — APPROVED** [review]
/lgtm
/approve

**Pull request merged**